libusb/libusbi.h

```diff
@@ -44,6 +44,17 @@
 #define static_assert(cond, msg) _Static_assert(cond, msg)
 #endif
 
+/* https://clang.llvm.org/docs/ThreadSafetyAnalysis.html */
+#ifdef __clang
```

Comment:
I believe this is how it is suggested by `ThreadSafetyAnalysis.html`
```suggestion
#if defined(__clang__) && (!defined(SWIG))
```